sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/samples/Sample2_DetectLastPointAnomaly.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.AI.AnomalyDetector.Models;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.AnomalyDetector.Tests.Samples
+{
+    public partial class AnomalyDetectorSamples : SamplesBase<AnomalyDetectorTestEnvironment>
+    {
+        [Test]
+        public async Task DetectLastPointAnomaly()
+        {
+            //read endpoint and apiKey
+            string endpoint = TestEnvironment.Endpoint;
```

Comment:
Let's add `Snippet:CreateAnomalyDetectorClient` and `Snippet:ReadSeriesData` regions here as well.

Reply:
Similarly for sample-3.